src/js/entities-service/entities/forms.js

```diff
@@ -36,9 +36,14 @@ const _Model = BaseModel.extend({
     return get(this.get('options'), 'beforeSubmit', defaultBeforeSubmit);
   },
   getWidgets() {
-    const widgets = get(this.get('options'), 'widgets', []);
+    const formWidgets = get(this.get('options'), 'widgets', { fields: [], widgets: [] });
```

Comment:
You don't really need the defaults on this `get`

Reply:
True, just tested it and everything works without the defaults there.